app/src/main/res/values/strings.xml

```diff
@@ -271,7 +271,7 @@
 
     <!-- Crashes -->
     <!-- Title text displayed on the tab crash page. This first parameter is the name of the application (For example: Fenix) -->
-    <string name="tab_crash_title">Sorry. %1$s can’t get that tab back</string>
+    <string name="tab_crash_title">Sorry. %1$s can’t load that page.</string>
```

Comment:
Now that we're exporting strings, we cannot change existing strings. We have to create new ones and delete the old ones.

Reply:
Didn't know we had started this process. Thanks for the heads up!